doc/lua_api.md

```diff
@@ -9508,14 +9514,20 @@ in active mapblocks.
 
     interval = 10.0,
     -- Operation interval in seconds
+    -- This value can be changed by `core.change_abm` call.
+    -- A negative interval value means that ABM is disabled.
+    -- The counter is running for disabled ABM as well. So it is recommended to use a negative value of normal interval to disable it.
+    -- If the interval is reduced to 50% or more, it can happen that ABM will be triggered in a few following server steps repeatedly.
```

Comment:
this sounds like documenting a bug. why not just fix this?

Reply:
Can't we have a cleaner API for disabling ABMs? Setting interval to `math.huge` would make sense for example, or a separate API function.

Reply:
I updated it, now chance set to 0 means ABM is disabled.

By our documentation, chance 0 is now an invalid value. So should not be used by mods.

Reply:
`chance` of `0` also makes sense :+1: